pages/common/aws-rds.md

```diff
@@ -0,0 +1,37 @@
+# aws rds
+
+> CLI for AWS RDS.
```

Comment:
What's "AWS RDS"? it might be work expanding on the RDS bit here, since AWS have so many services it's difficult for even an expert to remember them all.

Reply:
```suggestion
> CLI for AWS Relational Database Service.
```